### DISCUSSION TITLE
[Feature] Change a "regular" ticket to an internal ticket

### DISCUSSION THREAD

**Discussion by blu-IT** [YetiForceCompany/YetiForceCRM#14190] [Feature improvement]
### Describe the solution you'd like
I have some e-mail accounts that trigger a new ticket in YF. Some of these tickets do not belong to a Account/Organization/Customer but are INTERNAL tickets and I would like to change/transfer/migrate them to an internal (company) ticket.

### Describe alternatives you've considered
I could add my own company as an Account/Organization/Customer to solve this problem and assign these tickets then, but that would rend the funcion of "internal tickets" (where already a lot of internal tickets are stored) useless.

I am using YF 5.3. SF 1

Thanks!

Regards

**Comment by bpabiszczak** [comment]
Why doesn't the mail scanner decide this? Just add a good rule.

**Comment by blu-IT** [comment]
> Why doesn't the mail scanner decide this

You mean the YF mail scanner? Unfortunately I just found the actions to take there when a new mail comes into a mailbox.
But when the mail is in the "wrong" mailbox or if it is for example "support@mycompany.com" for external customers and 10 % of the received mails are only internal tickets, I don't know how to handle this. 
So the possibility to turn a "regular ticket" into an "internal ticket" would be nice, when there is only one mailbox available for example or someone send to the wrong box.
Letting the mailscanner decide would assume I have some criteria to filter, match and decide. Like a subject with "support-int" oder "support-ext".
But as I said, I even don't know where I could tell YF to take further actions then open a ticket and assign it somebody.
